collectors/logsmanagement.plugin/plugin_logsmanagement_docker_ev.c

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+#include "plugin_logsmanagement_docker_ev.h"
+
+void docker_ev_chart_init(struct File_info *p_file_info, struct Chart_meta *chart_meta){
+    chart_meta->chart_data_docker_ev = callocz(1, sizeof (struct Chart_data_docker_ev));
+    chart_data_docker_ev_t *chart_data = chart_meta->chart_data_docker_ev;
+
+    /* Number of lines - initialise */
+    chart_data->st_lines = rrdset_create_localhost(
+            (char *) p_file_info->chart_name
+            , "lines parsed"
+            , NULL
+            , "lines parsed"
+            , NULL
+            , "Log lines parsed"
+            , "lines/s"
+            , "logsmanagement.plugin"
+            , NULL
+            , NETDATA_CHART_PRIO_LINES
+            , p_file_info->update_every
+            , RRDSET_TYPE_AREA
+    );
+    // TODO: Change dim_lines_total to RRD_ALGORITHM_INCREMENTAL
+    chart_data->dim_lines_total = rrddim_add(chart_data->st_lines, "Total lines", NULL, 1, 1, RRD_ALGORITHM_ABSOLUTE);
+    chart_data->dim_lines_rate = rrddim_add(chart_data->st_lines, "New lines", NULL, 1, 1, RRD_ALGORITHM_INCREMENTAL);
+
+    /* Docker events type - initialise */
+    if(p_file_info->parser_config->chart_config & CHART_DOCKER_EV_TYPE){
+        chart_data->st_dock_ev_type = rrdset_create_localhost(
+                (char *) p_file_info->chart_name
+                , "events type"
+                , NULL
+                , "docker events"
+                , NULL
+                , "Events type"
+                , "events types/s"
```

Comment:
the same here, please, take a look in all your `units`.

Reply:
Done!